### PR TITLE
Drop cols via usecols arg on read_csv

### PR DESCRIPTION
**Pull request by rdlester**
Will prevent unused columns from ever being loaded into memory beyond during parsing, hopefully improving memory usage and time.

(I didn't verify every script still worked, please double check.)

### CODE REVIEW

**Comment by rdlester** [comment]
I checked that the scripts all work.

File: JPF/utils.py [hunk 1]
```diff
@@ -134,10 +134,11 @@ def hpd_csv2sql(description, input_csv_url, sep_char,\
             return
 
         # stream csv into chunked dataframes
-        log.info("Streaming CSV from {} ...".format(input_csv_url))
+        log.info("Streaming {} CSV fields from {} ...".format(
+            len(keep_cols), input_csv_url))
 
         csv_chunks = pd.read_csv(input_csv_url, sep=sep_char, dtype=dtype_dict,\
-                encoding='utf8', chunksize=csv_chunk_size)
+                encoding='utf8', chunksize=csv_chunk_size, usecols=keep_cols)
```
Comment (chrishenry) [inline JPF/utils.py:141]: Sweet.

For those following along, here's the docs for [pandas.read_csv](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_csv.html)

Comment (jpfreeley) [reply JPF/utils.py:141]: awesome .. i should have read those docs weeks ago .. at least I made it easy to utilize the usecols ... does it matter that my keep_cols is after lowered slugified cols?

Comment (rdlester) [reply JPF/utils.py:141]: Yeah, unfortunately - note the other file changes in this PR.

**Pull request merged**